frontend/src/components/team_calendar/ScheduleAddModal/ScheduleAddModal.tsx

```diff
@@ -172,25 +173,31 @@ const ScheduleAddModal = (props: ScheduleAddModalProps) => {
               </Text>
             </Button>
           </S.ConvenientContainer>
-          <S.DescriptionDiv $isDescription={isDescription}>
-            <S.DescriptionTextarea
-              rows={1}
-              placeholder="메모를 작성해주세요.(최대 100자)"
-              value={schedule.description}
-              ref={descriptionInputRef}
-              onChange={handleDescriptionInput}
-              required
-            />
-            <S.WarnDiv>
-              {!isDescriptionMaxLength ? (
-                <Text size="xs">({schedule.description.length} / 100자)</Text>
-              ) : (
-                <Text size="xs" css={S.errorText}>
-                  최대 100자까지 입력가능합니다.
-                </Text>
-              )}
-            </S.WarnDiv>
-          </S.DescriptionDiv>
+          {isDescription && (
+            <>
+              <S.DescriptionTextarea
+                rows={1}
+                placeholder={`메모를 작성해주세요.(최대 ${SCHEDULE_DESCRIPTION_MAX_LENGTH}자)`}
+                value={schedule.description}
+                ref={descriptionInputRef}
+                onChange={handleDescriptionInput}
+              />
+              <S.WarnDiv>
+                {!isDescriptionMaxLength ? (
+                  <Text size="xs">
+                    ({schedule.description.length} / $
+                    {SCHEDULE_DESCRIPTION_MAX_LENGTH}자)
+                  </Text>
```

Comment:
의도했던 결과는 아니지?

```suggestion
                  <Text size="xs">
                    {`(${schedule.description.length} / ${SCHEDULE_DESCRIPTION_MAX_LENGTH}자)`}
                  </Text>
```
![image](https://github.com/user-attachments/assets/514bdb7f-8c7a-438c-bbe0-9ed77866a09e)